boards/arm/udoo_neo_full_m4/udoo_neo_full_m4.yaml

```diff
@@ -18,5 +18,3 @@ supported:
   - uart
   - gpio
   - counter
-testing:
-  default: true
```

Comment:
What does it mean to be a default testing platform?

Reply:
@MaureenHelm 
when you run sanitycheck without any arguments, boards marked as default get full coverage. This is relevant to CI running on PR jobs, this way we can control what to run and keep the CI times sane.

Reply:
Do boards not marked as default run on nightly builds?

Reply:
yes, everything runs in the nightly build.

default is set to a representative set of boards, mostly qemu:

```
boards/arc/em_starterkit/em_starterkit_em7d.yaml:  default: true
boards/arm/frdm_k64f/frdm_k64f.yaml:  default: true
boards/arm/mps2_an385/mps2_an385.yaml:  default: true
boards/arm/nrf51_pca10028/nrf51_pca10028.yaml:  default: true
boards/arm/udoo_neo_full_m4/udoo_neo_full_m4.yaml:  default: true
boards/nios2/qemu_nios2/qemu_nios2.yaml:  default: true
boards/posix/native_posix/native_posix.yaml:  default: true
boards/riscv32/qemu_riscv32/qemu_riscv32.yaml:  default: true
boards/x86/qemu_x86/qemu_x86.yaml:  default: true
boards/x86/qemu_x86/qemu_x86_nommu.yaml:  default: true
boards/x86_64/qemu_x86_64/qemu_x86_64.yaml:  default: true
boards/xtensa/qemu_xtensa/qemu_xtensa.yaml:  default: true
boards/xtensa/xt-sim/xt-sim.yaml:  default: true
```